tests/README.md

```diff
@@ -80,16 +80,15 @@ In addition to standard go tests parameters, the following custom parameters are
  Parameter | Description | Possible values | Default
  --- |--- | --- | ---
 rook_platform| platform rook needs to be installed on  | kubernetes | kubernetes
-k8s_version  | version of Kubernetes to be installed  | v1.5+  | v1.7.5
+k8s_version  | version of Kubernetes to be installed  | v1.6+  | v1.8.5
 rook_image | rook image name to be installed | valid image name | rook/rook
 toolbox_image | toolbox image name to be installed | valid image name | rook/toolbox
 skip_install_rook | skips installing rook (if already installed) | true or false  | false
-load_parallel_runs | performs concurrent operations (optional used for load test) | any number | 20
-
-If the `install_rook` flag is set to false, then all the other flags are ignored
-and tests are run without rook being installed and setup. Use this flag to run tests against
-a pre-installed/configured rook.
-
+load_parallel_runs | performs concurrent operations (load test only) | any number | 20
+load_volumes | number of volumes ( block,object store or shared file systems) to be created during load test (load test only)| >1 | 1
+load_time | number of seconds load test run for (load test only) | >1 | 1800
```

Comment:
`number of seconds to run (load test only)`

Reply:
fixed.